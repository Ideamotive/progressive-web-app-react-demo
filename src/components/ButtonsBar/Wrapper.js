import React from 'react';
import styled from 'styled-components';

export default styled.div`
    height: 100%;
    position: relative;
    display: flex;
    justify-content: space-around;
    align-items: center;
  `;