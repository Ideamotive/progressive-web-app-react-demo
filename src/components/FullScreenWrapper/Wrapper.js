import React from 'react';
import styled from 'styled-components';

export default styled.div`
    position: fixed;
    width: 100%;
    height: 100%;
    left: 0;
    top: 0;
    z-index: 2;
  `;