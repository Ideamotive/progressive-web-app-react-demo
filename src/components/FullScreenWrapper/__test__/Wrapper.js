import React from 'react';
import * as renderer from 'react-test-renderer';

import Wrapper from '../Wrapper';

describe('<Wrapper/> component', () => {
  test('should match to snapshot', () => {
    const tree = renderer
      .create(
        <Wrapper>
          {'test'}
        </Wrapper>,
      )
      .toJSON();
    expect(tree).toMautchSnapshot();
  });
});

