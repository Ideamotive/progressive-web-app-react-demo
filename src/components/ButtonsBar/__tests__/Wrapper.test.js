import React from 'react';
import * as renderer from 'react-test-renderer';

import Wrapper from '../Wrapper';

describe('<Wrapper/> component', () => {
  test('should match to snapshot', () => {
    const tree = renderer
      .create(<Wrapper
        backgroundColor={'#fff'}
      >
        {'test'}
      </Wrapper>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

