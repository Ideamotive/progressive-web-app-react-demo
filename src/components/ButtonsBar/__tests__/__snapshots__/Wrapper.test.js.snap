// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper/> component should match to snapshot 1`] = `
<div
  className="sc-bdVaJa ksoPgM"
>
  test
</div>
`;
